src/interfaces/dto/recipeDtos.ts

```diff
@@ -1,15 +1,11 @@
-import { IUser } from '../user';
-
 export interface createRecipeDto {
```

Comment:
interfaces / classes should be PascalCase, so:

`export interface CreateRecipeDto`